<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyWindow</class>
 <widget class="QDialog" name="KeyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Virtual keyboard</string>
  </property>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEncPUSH">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PUSH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRIGHT">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+/ &gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnESC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ESC</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLEFT">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt; / -</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
